xmake/modules/private/action/run/runenvs.lua

```diff
@@ -58,6 +58,14 @@ function _make_runpath_on_windows(target)
                 end
                 insert_target(dep)
             end
+            for _, toolchain in ipairs(target:toolchains()) do
+                local runenvs = toolchain:get("runenvs")
+                if runenvs then
+                    for _, runenv in pairs(runenvs) do
```

Comment:
Do I need to split runenv by the env path separator (if there are more than one value) ?